<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContrastWidget</class>
 <widget class="QWidget" name="ContrastWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBoxCalculation">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>230</y>
     <width>341</width>
     <height>141</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>计算结果</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="ContrastCalc">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>计算</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_AutoSet">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>自动设置</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>50</y>
     <width>341</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>数据范围</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <widget class="QLineEdit" name="HighThreshold">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>40</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>---</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="LowThreshold">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>110</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>---</string>
    </property>
    <property name="cursorPosition">
     <number>3</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>41</y>
      <width>54</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>上限</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>54</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>下限</string>
    </property>
   </widget>
   <widget class="QLabel" name="ModelContrastLabel">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>75</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>75</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>对比度</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
